packages/react-components/stable-review/react-components.api.md

```diff
@@ -358,6 +360,49 @@ export interface DevicesButtonStyles extends ControlBarButtonStyles {
     menuStyles?: Partial<DevicesButtonContextualMenuStyles>;
 }
 
+// @beta
+export const Dialpad: (props: DialpadProps) => JSX.Element;
+
+// @beta
+export interface DialpadButtonsType {
+    // (undocumented)
+    primaryContent: string;
+    // (undocumented)
+    secondaryContent?: string;
+}
+
+// @beta
+export interface DialpadProps {
+    // (undocumented)
+    dialpadButtons?: DialpadButtonsType[][];
+    // (undocumented)
+    strings?: DialpadStrings;
+    // (undocumented)
+    styles?: Partial<DialpadStyles>;
+}
+
+// @beta
+export interface DialpadStrings {
+    // (undocumented)
+    defaultText: string;
+    // (undocumented)
+    errorText: string;
+}
+
+// @beta
+export interface DialpadStyles {
+    // (undocumented)
+    button?: IButtonStyles;
+    // (undocumented)
+    digit?: IStyle;
```

Comment:
nit: consider keeping naming consistent with the DialpadButtonsType, so primaryContent and secondaryContent, instead of digit and subDigit

Reply:
updated